django/cantusdb_project/main_app/urls.py

```diff
@@ -190,8 +190,8 @@
     ),
     path(
         "chants/",
-        views.redirect_chant_list,
-        name="redirect-chant-list",
+        views.redirect_chants,
+        name="redirect-chants",
```

Comment:
Do these tests still pass?

It looks like we use `reverse("redirect-chant-list")`...

https://github.com/DDMAL/CantusDB/blob/fe15edc7106369eef622fe7df5e585a95bf6cfb7/django/cantusdb_project/main_app/tests/test_views.py#L710-L727

Reply:
A quick text search of the rest of the repo looks like we don't use the old name anywhere else...does that make sense to you?

Reply:
Thanks for catching this!